server/service/src/programs/schemas/contact_trace.json

```diff
@@ -100,17 +100,27 @@
         "extension": {
           "type": "object"
         },
-        "status": {
-          "$ref": "#/definitions/ContactTraceStatus",
-          "description": "Current status"
+        "location": {
```

Comment:
I couldn't save the contacts - and then noticed that some new bits were missing. I updated using this JSON schema but then had issues because my ui_schema was out of date.

so, I took the ui_schema from the programschemas repo, which caused more problems.

I've now taken the JSON from that repo as well, and things are working. But the JSON differs to this file! Does this one require updating?

Reply:
Hmm its working here and copied over the latest version and there is no diff. Maybe run `./build_all.sh` to generate the latest json schemas...